infrastructure/src/main/java/org/corfudb/infrastructure/IInvokeCheckpointing.java

```diff
@@ -0,0 +1,7 @@
+package org.corfudb.infrastructure;
+
+public interface IInvokeCheckpointing {
```

Comment:
Please don't use 'I' prefix, that is very bad practice

Reply:
I tried to follow the interface naming convention based on our repo. Like the IHoleFillPolicy, IClient, IServerRouter, etc. Changing this might hurt the pattern for interfaces in the repo right?